notebooks/00_quick_start/run_notebook_on_aml.ipynb

```diff
@@ -0,0 +1,296 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<i>Copyright (c) Microsoft Corporation. All rights reserved.</i>\n",
+    "\n",
+    "<i>Licensed under the MIT License.</i>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Run Notebook on AML\n",
+    "## Introduction to Azure Machine Learning  \n",
```

Comment:
This may occur at several places, and it would be good we put it some where so that different workloads share it in common.

Reply:
You're right, we have same instruction in other notebooks as well. There was a discussion on this. We want at least the critical information to use AzureML such as subscription creation and compute resource creation to stay in the same notebook, so that users don't click away and never come back. However, if there's a way to reference same instruction from multiple notebooks, I'll refactor.